CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Remove error message when reloading an empty config
 - Allow disabling URL launching by setting the value of `mouse.url.launcher` to `None`
 - Corrected the `window.decorations` config documentation for macOS
+- Scale IME position by DPI
```

Comment:
This will likely not tell the user at all what actually got fixed, I'd recommend putting it in simpler terms like this:

```
- Fix IME position on hidpi displays
```

Reply:
HiDPI even.

Reply:
Done. Though this would also affect low DPI displays like projectors, but the code only ever mentions high DPI so this is consistent.